src/modules/launcher/Plugins/Microsoft.Plugin.Uri/UriHelper/UriResolver.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Plugin.Uri.Interfaces;
+
+namespace Microsoft.Plugin.Uri.UriHelper
+{
+    public class UriResolver : IUrlResolver
+    {
+        public bool IsValidHost(System.Uri uri)
```

Comment:
Does the Uri.TryParse return false if the host name is invalid? If so I think this method is unnecessary.   Otherwise we should probably use or wrap [Uri.CheckHostName](https://docs.microsoft.com/en-us/dotnet/api/system.uri.checkhostname?view=netcore-3.1)

Reply:
I think the URI builder will throw if the hostname is invalid and we don't need this check.

Reply:
The parses will only contain valid hostnames/ipadresses.

This Resolver was originally used as a DNS lookup check to see if the url is indeed resolvable, but it was decided to let the browser handle this. So this can be removed / or keep it and  look in the future for some additional validation checks (It's a valid hostname, but we don't want the result in the list.